app/admin/company/[company_name]/page.tsx

```
@@ -1,14 +1,95 @@
-        <Suspense fallback={<div>Loading company details...</div>}>
-            <CompanyDetailContent companyName={companyName} />
-        </Suspense>
+        <div className="mx-auto">
+            <Suspense fallback={<p>Loading Company Detail</p>}>
+                <p className="text-5xl mb-3">{name}</p>
+                <p>Company Type: {type}</p>
+                <p>Company Size: {size} people</p>
+                <p>Do they already have an intern? Yes <Checkbox checked={haveIntern === 'yes' ? true : false} /> No <Checkbox checked={haveIntern === 'no' ? true : false} /></p>
+                <p>Programming Languages:</p>
+                <p className="text-md">{program_languages.map((lang, index) => (
+                    <ul>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)